src/Ojs/ApiBundle/Handler/JournalArticleHandler.php

```diff
@@ -148,7 +151,6 @@ private function processForm(Article $entity, array $parameters, $method = "PUT"
         }
         if ($form->isValid()) {
             $entity->setCurrentLocale('en');
-            $entity->setStatus(ArticleStatuses::STATUS_PUBLISH_READY);
```

Comment:
this line change looks like correct touch. but, if api user not specifies the status what happens. we can discuss this issue. Anyway for other changes i will merge this commit